src/components/MainContent.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import TaskForm from "./TaskForm";
+import TaskCard from "./TaskCard";
+
+const MainContent = () => {
+  const [tasks, setTasks] = useState([]);
+  const [taskInput, setTaskInput] = useState("");
+  return (
+    <div className="flex justify-center items-center min-h-screen text-gray-800 p-6">
+      <div className="max-w-2xl mx-auto">
+        <h1 className="text-slate-300 text-center text-5xl font-bold">Todos</h1>
+        <div className="mt-10 space-y-6">
+          <TaskForm
+            tasks={tasks}
+            setTasks={setTasks}
+            taskInput={taskInput}
+            setTaskInput={setTaskInput}
+          />
+          <div className="flex flex-col gap-4">
+            {tasks.map((task, index) => (
```

Comment:
Consider this few lines to know what you have to give to your key prop value from the below url: 
[https://legacy.reactjs.org/docs/lists-and-keys.html#:~:text=Keys%20should%20be%20given%20to,list%20item%20among%20its%20siblings.](url)

> Rules of keys 
> Keys must be unique among siblings. However, it’s okay to use the same keys for JSX nodes in different arrays.
> Keys must not change or that defeats their purpose! Don’t generate them while rendering.

consider those 2 lines in the document from the new react